README.md

```diff
@@ -49,7 +51,9 @@
 ## Internationalization (I18n)
 Please review [Terra's Internationalization documentation](https://engineering.cerner.com/terra-ui/#/getting-started/terra-ui/internationalization) for more information. Included are directions on consumption and how internationalization is setup.
 
-## Contributing
+<h2 id="contributing">
```

Comment:
Do we need to do this for Status and Internationalization as well?

Reply:
We only need ids for the headers that we have anchors to. But we should probably have anchors to those as well, I'll update.